### PR TITLE
Add `useValues()` hook 🎣

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

**Comment by wcandillon** [comment]
@dsznajder what do you think? 🤔

File: README.md [hunk 1]
```diff
@@ -245,6 +245,22 @@ contains(values: Node[], value: Node) => Node
...
+For instance the code snippet below:
+
+```
+const [toggle, state] = useValues([0, State.UNDETERMINED], []);
+```
+
+is a shortcut for
+
+```
+const [toggle state] = useMemo(() => [new Value(0), new Value(State.UNDETERMINED)], []);
```
Comment (dsznajder) [inline README.md]: I believe it's:
```suggestion
const [toggle state] = useMemoOne(() => [new Value(0), new Value(State.UNDETERMINED)], []);
```
As useMemo can drop those at some point :)

**Comment by wcandillon** [comment]
@dsznajder Great idea! We can add support for both :)

**Comment by dsznajder** [comment]
@wcandillon great 🎉

**Pull request merged**